### PR TITLE
Combat-trainer clutter management

### PR DESCRIPTION
**Pull request by drboldwin**

### CODE REVIEW

**Comment by drboldwin** [comment]
Hmm was gonna submit those seperately - base.yaml I just added a dump_junk: false - but take a look at it because I am seeing an odd waggle addition/subtraction that I didn't do.  Maybe this was updated between me forking and submitting?

**Comment by rpherbig** [comment]
The `waggle_sets` line looks like a newline was added to the end of the file. Don't worry about it.

File: combat-trainer.lic [hunk 1]
```diff
@@ -197,9 +197,18 @@ class LootProcess
 
     @loot_specials = settings.loot_specials
     echo("  @loot_specials: #{@loot_specials}") if $debug_mode_ct
+    @dump_junk = settings.dump_junk
+    @dump_delay = 300 # in seconds
```
Comment (rpherbig) [inline combat-trainer.lic]: Do we need the delay stored as a variable if it is only used in one place? @rcuhljr @kamarcum thoughts?

The same thing probably applies to `@loot_delay` on line 191.

File: combat-trainer.lic [hunk 2]
```diff
@@ -197,9 +197,18 @@ class LootProcess
 
     @loot_specials = settings.loot_specials
     echo("  @loot_specials: #{@loot_specials}") if $debug_mode_ct
+    @dump_junk = settings.dump_junk
```
Comment (rpherbig) [inline combat-trainer.lic]: Please add an echo that outputs the state of this setting (see line 199) for an example.

File: combat-trainer.lic [hunk 3]
```diff
@@ -197,17 +197,16 @@ class LootProcess
 
     @loot_specials = settings.loot_specials
     echo("  @loot_specials: #{@loot_specials}") if $debug_mode_ct
+    
     @dump_junk = settings.dump_junk
-    @dump_delay = 300 # in seconds
-    @dump_timer = Time.now - @dump_delay
+    echo("  @dump_junk: #{@dump_junk}") if $debug_mode_ct
```
Comment (drboldwin) [inline combat-trainer.lic]: Believe this is what you wanted.

File: combat-trainer.lic [hunk 4]
```diff
@@ -197,17 +197,16 @@ class LootProcess
...
+    @dump_timer = Time.now - 300
```
Comment (drboldwin) [inline combat-trainer.lic]: got rid of dump_delay and just put 300 in there

File: combat-trainer.lic [hunk 5]
```diff
@@ -197,17 +197,16 @@ class LootProcess
...
   end
 
   def execute(game_state)
-    if (Time.now - @dump_timer > @dump_delay) && @dump_junk
-      if DRRoom.room_objs.count >= 13
-        fput "DUMP JUNK"
-        @dump_timer = Time.now
-      end
+    if (Time.now - @dump_timer > 300) && @dump_junk && DRRoom.room_objs.count >= 13
```
Comment (drboldwin) [inline combat-trainer.lic]: got rid of the embedded if statement and made it one large one.  Also adjusted for the lack of dump_delay

**Pull request merged**